src/model/AirLoopHVACOutdoorAirSystem.hpp

```diff
@@ -148,7 +151,7 @@ namespace model {
     bool setControllerOutdoorAir(const ControllerOutdoorAir& controllerOutdoorAir);
 
     /** Reimplemented from HVACComponent. **/
-    boost::optional<AirLoopHVAC> airLoop() const;
+    boost::optional<AirLoopHVAC> airLoop() const;  // TODO: this shouldn't exist!!!
```

Comment:
Not breaking API, but to consider for a breaking release. I made it forward to the Impl_::airLoopHVAC() like it should have been